version.txt

```diff
@@ -1,81 +1,79 @@
 v1.9.0-dev
-AvianBodySize.json
-FishParasiteHosts.json
-Steppe_plants_2013.json
-CRC_avianbodymass.py
-EltonTraits.json
-leaf_herbivory.json
-gentry.py
-CForBioData.json
-ArcticBreedBird.json
-PortalMammals.json
-leda.py
-Anderson2011.json
-Pantheria.json
-bbs50stop.py
-PhytoplankonBiovolume.json
-eBird_observation.py
-MoM2003.json
-SonoranPerennials.json
-MammalLH.json
-HomeRanges.json
-MammalSuperTree.py
-FrayJorge.json
-gwdd.py
-vertnet.py
-MarineSize.json
-EA_petraitis2008.py
-McGlinn2010.json
-MiscAbundanceDB.json
-TreeWesternGhats.json
-fia.py
-Woods2009.json
-MCDB.json
-DelMoral2010.json
-npn.py
-Matter2014.json
-bioclim_2pt5.py
-Palmer2007.json
-Adler2007.json
-Ramesh2010.json
-MammalMR2010.json
-PlantTaxonomy.json
-MammalDIET.json
```

Comment:
@goelakash, Line [43 MammalDIET.json](https://github.com/weecology/retriever/pull/635/files#diff-5fb01c6f8b54d96c62af3415c51ffad3L43) and line [54 MammalDiet.script](https://github.com/weecology/retriever/pull/635/files#diff-5fb01c6f8b54d96c62af3415c51ffad3L54) should these be having the same name ? Wondering if Capitals make a difference